services/device-registry-base/src/main/java/org/eclipse/hono/service/management/device/Device.java

```diff
@@ -33,6 +34,7 @@
  * Device Information.
  */
 @JsonInclude(value = Include.NON_NULL)
+@JsonIgnoreProperties({ "status" })
```

Comment:
Do we need this here? FMPOV `Device` has no property named _status_ defined. In order to include the _status_ we have a new class `DeviceWithStatus`. If needed, then `RegistryManagementConstants.FIELD_STATUS` could be used instead of the string literal `"status"`.

Reply:
The reason why I added this is: if a client sends an update via a PUT request and uses copy&paste from a previous GET request to get the body's content will have the status property in it. In order to make this convenient and not requiring the status property to be removed I set it to ignore here.

I'll use the literal as suggested.